styles/colors-silicon-night.less

```diff
@@ -230,6 +230,31 @@
 @agate-switch-focus-color: black;
 @agate-switch-label-color: @agate-accent;
 
+// ThumbnailItem
+// ---------------------------------------
+@agate-thumbnailitem-focus-text-color: @agate-white;
+@agate-thumbnailitem-subcontent-selected-text-color: @agate-white;
+@agate-roundthumbnailitem-border-color: @agate-white;
+
+// Toggle Colors
+// ---------------------------------------
+@agate-togglebutton-indicator-off-bg-color: black;
+@agate-togglebutton-indicator-on-bg-color: @agate-highlight;
+@agate-togglebutton-indicator-off-border-color: @agate-button-border-color;
+@agate-togglebutton-indicator-on-border-color: @agate-accent;
+@agate-togglebutton-indicator-off-focus-bg-color: @agate-togglebutton-indicator-off-bg-color;
+@agate-togglebutton-indicator-on-focus-bg-color: @agate-togglebutton-indicator-on-bg-color;
+@agate-togglebutton-indicator-off-focus-border-color: @agate-togglebutton-indicator-on-focus-border-color;
+@agate-togglebutton-indicator-on-focus-border-color: @agate-white;
+@agate-togglebutton-indicator-disabled-off-bg-color: @agate-darker-gray;
+@agate-togglebutton-indicator-disabled-on-bg-color: @agate-togglebutton-indicator-disabled-off-bg-color;
+@agate-togglebutton-indicator-disabled-off-border-color: #6c6c6c;
```

Comment:
Let's define #6c6c6c to @agate-dim-gray referred from https://www.htmlcsscolor.com/hex/6C6C6C.